tests/api/api/test_functions.py

```diff
@@ -16,15 +16,19 @@
 import mlrun.artifacts.dataset
 import mlrun.artifacts.model
 import mlrun.errors
+import tests.api.api.utils
 import tests.conftest
 
+PROJECT = "project-name"
+
 
 def test_build_status_pod_not_found(db: Session, client: TestClient):
+    tests.api.api.utils.create_project(client)
```

Comment:
Either you're passing the `PROJECT` define here to the `create_project` or resolve the name from the returned value from this function
Now you're just counting on that the `PROJECT` here is equal to the `create_project` default